src/main/java/com/redhat/devtools/intellij/knative/ui/CreateServiceDialog.java

```diff
@@ -92,6 +96,31 @@ private void initEditor(String namespace) {
             logger.warn(e.getLocalizedMessage(), e);
         }
         editor = new PsiAwareTextEditorImpl(project, new LightVirtualFile("service.yaml", content), TextEditorProvider.getInstance());
+        editor.getEditor().getDocument().addDocumentListener(createEditorListener());
+    }
+
+    private com.intellij.openapi.editor.event.DocumentListener createEditorListener() {
+        return new com.intellij.openapi.editor.event.DocumentListener() {
+            @Override
+            public void documentChanged(com.intellij.openapi.editor.event.DocumentEvent event) {
+                try {
+                    String nameInYAML = YAMLHelper.getStringValueFromYAML(editor.getEditor().getDocument().getText(), new String[] { "metadata", "name" });
```

Comment:
Maybe good to have those YAML paths as constants as they are duplicated several times